src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/service/ReferralService.kt

```diff
@@ -58,6 +58,12 @@ class ReferralService(val repository: ReferralRepository) {
       }
     }
 
+    update.usingRarDays?.let {
+      if (it && update.maximumRarDays == null) {
+        errors.add(FieldError(field = "usingRarDays", error = Code.CONDITIONAL_FIELD_MUST_BE_SET))
```

Comment:
We could use a custom code here, for example `rar_days_must_be_set`. Do we need to use generic ones?

Reply:
this does mean there any many more error codes for the front end to manage though. isn't the field info with this message sufficient to convey what the issue is?